docker/web/src/main/docker/Dockerfile

```diff
@@ -1,11 +1,29 @@
-FROM deephaven/nginx-base:local-build
+FROM deephaven/nginx-noroot-base:local-build
+
+# In order to run without root, we start as root, install sudo w/ sudo rules, then switch to nginx user (id 101)
+USER 0
+
+# We want to install sudo, and start the sudo service (update policy-rc.d to exit 0, instead of 101)
+RUN ls -la /usr/sbin/policy-rc.d ; \
```

Comment:
convention is to prefix any multi-line RUN commands with `set -eux` to avoid surprises when one command fails, then the `&&`s aren't required. Alternatively, move contents into a `.sh` file, and start that with `set -eux`, invoke that directly (though that involves copying it first, etc)

Reply:
fixed